core/frontend/src/test/utils/webpack.config.js

```diff
@@ -55,6 +55,10 @@ function createConfig(shouldInstrument) {
             env[key] = JSON.stringify(process.env[key]);
             return env;
           }, {}),
+      }),
+      // Disable code-splitting for tests - see dynamic import in ApproximateTerrainHeights.ts.
```

Comment:
@wgoehrig this resolves the following error  when running certa - look reasonable?
```
  1) ApproximateTerrainHeights
       test correct heights:
     ChunkLoadError: Loading chunk 0 failed.
(error: http://localhost:3000/0.bundled-tests.js)
      at Function.requireEnsure [as e] (webpack/bootstrap:105:1)
      at ApproximateTerrainHeights.initialize (D:\j2\s\imodeljs\core\frontend\src\ApproximateTerrainHeights.ts:42:37)
      at Context.call (D:\j2\s\imodeljs\core\frontend\src\test\tile\map\ApproximateTerrainHeights.test.ts:21:26)
      at callFn (@/lib/runnable.js:366:21)
      at Test.run (@/lib/runnable.js:354:5)
      at Runner.runTest (@/lib/runner.js:677:10)
      at fn (@/lib/runner.js:801:12)
      at next (@/lib/runner.js:594:14)
      at fn (@/lib/runner.js:604:7)
      at next (@/lib/runner.js:486:14)
```

Reply:
Reverted.